core/types/block.go

```diff
@@ -36,12 +36,13 @@ import (
 )
 
 var (
-	EmptyRootHash  = common.HexToHash("56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421")
-	EmptyUncleHash = RlpHash([]*Header(nil))
-	EmptyBodyHash  = common.HexToHash("51e1b9c1426a03bf73da3d98d9f384a49ded6a4d705dcdf25433915c3306826c")
-	big2e256       = new(big.Int).Exp(big.NewInt(2), big.NewInt(256), nil) // 2^256
-	hasher         = blake3.New(32, nil)
-	hasherMu       sync.RWMutex
+	EmptyRootHash     = common.HexToHash("56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421")
+	EmptyUncleHash    = RlpHash([]*Header(nil))
+	EmptyTerminusHash = RlpHash([]*Header(nil))
```

Comment:
These are only used in this file, so don't need to be public.

Reply:
not true. only emptyTerminusHash.  Will change.